tools/prereqs/setup_terraform.sh

```diff
@@ -30,8 +30,10 @@ cd $tmp_dir
 ostype=`os_type`
 if [ "$ostype" == "linux" ]; then
     arch="linux_amd64"
+    sudo apt-get install unzip
 elif [ "$ostype" == "macos" ]; then
     arch="darwin_amd64"
+    brew install unzip
```

Comment:
this looks good - maybe we can create a follow up issue to track that homebrew is a prereq to invoke `brew` - doesn't have to be in this PR.

https://brew.sh/

Reply:
Recorded here: https://github.com/microsoft/bedrock/issues/1248